conf/application.conf

```diff
@@ -205,8 +205,8 @@ metrics {
   enabled = false
 }
 
-google-tag-manager {
-  id = N/A
+tracking-consent-frontend {
+  gtm.container = "c"
```

Comment:
Is there a reason the this is now set to "c"over N/A?

Reply:
And if that's required then you'll also need to update [app-config-base](https://github.com/hmrc/app-config-base/blob/master/ers-checking-frontend.conf) with the new value

Reply:
should have said that the ticket got altered a tad, so im making changes currently. I will be changing the config too.
n/a was only used to exclude the gtm snippet on environments- no longer needed with tracking_consent_frontend